os/arch/arm/src/amebad/Kconfig

```diff
@@ -61,6 +61,30 @@ config AMEBAD_WIFI
 
 endmenu
 
+menu "Realtek RTL8721D BLE Support"
+
+config AMEBAD_BLE
+	bool "Enable Amebad BLE"
+	default n
+	select FTL_ENABLED
+
+config FTL_ENABLED
+	bool "Enable FTL"
+	default n
+	depends on AMEBAD_BLE
+
+config AMEBAD_BLE_PERIPHERAL
+	bool "Enable Amebad BLE Peripheral"
+	default n
+	depends on AMEBAD_BLE
+
+config AMEBAD_BLE_CENTRAL
```

Comment:
Don't you need to enable at least one functionality among peripheral and central? If it is, we should use choice for them.

Reply:
Yes. But it may need to enable more than one functionality. If choice is used, only one ca be selected.

Reply:
Ok.